### DISCUSSION TITLE
Sunshine API endpoints for managing connected clients

### DISCUSSION THREAD

**Discussion by User-green** [LizardByte/community#586] [Sunshine]
### Select Topic Area

Feature Request

### Body

I propose adding following API endpoints: 

GET /api/connections
Returns list of all currently connected clients as list of <connection ID>s.

GET /api/connections/{id}
Returns data on certain connection. Returns:
- time elapsed since connected
- time elapsed since last active
- source ip
- destanation ip 
- client public key

DELETE /api/connections/{id}
Requires:
- apikey
- Kill or detach? (true/false)
- Execute revert command? (true/false)
Kills selected connection. 

Admin password can be temporary used as apikey - untill proper apikey system created.

**Comment by User-green** [comment]
This will be useful for a number of reasons. Primary one - this will allow monitor switching scripts much finer control. Secondary one - if someone does something unauthorized, this API will allow to rapidly and automatically block them from assessing the system.

**Comment by swirle13** [comment]
This will also prove useful for my situation: streaming from Moonlight on a raspberry pi which sometimes disconnects the session for various reasons, when this occurs, Sunshine still thinks the connection is active and any subsequent attempts by Moonlight to reconnect and establish a session are met with timeouts and rejected connections. This is only fixed by restarting Sunshine.

I'd like to be able to know if there's a current session by the same device so I can have a script check, then disconnect the session, if it exists, and then attempt to connect again via Moonlight.
